aider/website/docs/usage.md

```diff
@@ -63,6 +63,16 @@ You'll get the best results if you think about which files need to be
 edited. Add **just** those files to the chat. Aider will include
 relevant context from the rest of your repo.
 
+## Read-only files
+
+You can also add files to the chat as "read-only" files. Aider can see these files for context, but can't edit them. This is useful for providing reference documentation, specifications, or examples of existing code that you don't want the AI to modify.
```

Comment:
should probably line-break these lines to follow the existing form of the usage.md.